third_party/libpg_query/grammar/statements/variable_show.y

```diff
@@ -21,10 +21,15 @@ VariableShowStmt:
 				n->is_summary = 1;
 				$$ = (PGNode *) n;
 			}
-		 | show_or_describe qualified_name
+		 | show_or_describe qualified_name opt_from
 			{
 				PGVariableShowStmt *n = makeNode(PGVariableShowStmt);
-				n->relation = $2;
+				if ($3) {
```

Comment:
It looks to me like "$2" is ignored if "$3" is set, what if we do `SHOW SOMETHING_ELSE FROM DB`? Can we at least verify that this is `SHOW TABLES FROM ...`?